dipy/tracking/fbcmeasures.pyx

```diff
@@ -0,0 +1,358 @@
+                fiber = fiber[0:self.streamline_length[i] - 1]
+                streamline_out.append(fiber)
+
+                rfbc_out.append(rfbc)
+
+                lfbc = lfbc[0:self.streamline_length[i] - 1]
+                lfbc_colors = np.transpose([fr(lfbc), fg(lfbc), fb(lfbc)])
+                color_out.append(lfbc_colors.tolist())
+
+        return streamline_out, color_out, rfbc_out
+    
+    @cython.wraparound(False)
+    @cython.boundscheck(False)
+    @cython.nonecheck(False)
+    @cython.cdivision(True)
+    cdef void compute(self, 
+                      py_streamlines,
+                      kernel,
+                      min_fiberlength,
+                      max_windowsize,
+                      num_threads=None,
+                      verbose=False):
+        """ Compute the fiber to bundle coherence measures for a set of 
+        streamlines.
+
+        Parameters
+        ----------
+        py_streamlines : list
+            A collection of streamlines, each n by 3, with n being the number of
+            nodes in the fiber.
+        kernel : Kernel object
+            A diffusion kernel object created from EnhancementKernel.
+        min_fiberlength : int
+            Fibers with fewer points than minimum_length are excluded from FBC computation.
+        max_windowsize : int
+            The maximal window size used to calculate the average LFBC region
+        num_threads : int
+            Number of threads to use for OpenMP.
+        verbose : boolean
+            Enable verbose mode.
+        """
+        cdef:
+            int num_fibers, max_length, dim
+            double [:, :, :] streamlines
+            int [:] streamlines_length
+            double [:, :, :] streamlines_tangent
+            int [:, :] streamlines_nearestp
+            double [:, :] streamline_scores
+            double [:] tangent
+            int line_id, point_id
+            int line_id2, point_id2
+            double score
+            double [:] score_mp
+            int [:] xd_mp, yd_mp, zd_mp
+            int xd, yd, zd, N, hn
+            double [:, :, :, :, ::1] lut
+            int threads_to_use = -1
+            int all_cores = openmp.omp_get_num_procs()
+
+        if num_threads is not None:
+            threads_to_use = num_threads
+        else:
+            threads_to_use = all_cores
+
+        if have_openmp:
+            openmp.omp_set_dynamic(0)
+            openmp.omp_set_num_threads(threads_to_use)
+        
+        # if the fibers are too short FBC measures cannot be applied, 
+        # remove these.
+        streamlines_length = np.array([len(x) for x in py_streamlines], 
+                                dtype=np.int32)
+        min_length = min(streamlines_length)
+        if min_length < min_fiberlength:
+            print("The minimum fiber length is 10 points. \
+                    Shorter fibers were found and removed.")
+            py_streamlines = [x for x in py_streamlines if len(x) >= min_fiberlength]
```

Comment:
By the way, here as well: 

    [x for x in py_streamlines if x.shape[0 >= min_fiberlength]

would be faster